### ISSUE TITLE
Is it possible to monitor temperature and humidity of HmIP-WTH-2?

### ISSUE THREAD

**Issue by joda123** [alexreinert/homematic_check_mk#5]
Hi Alex!
Thank you for providing homematic_check_mk - great thing!
homematic_check_mk provides several checks, e.g. an on board temperature, which is being displayed as a graph and I could rise a warn/error notification on it - great!

OMD[prod2]:~$ cmk -v ccu3-webui.jd.cretis.de | grep Temp
Temperature Zone 0   OK - 47.8 °C

Question: Is it possible to monitor a standard temp sensor like HmIP-WTH-2?
On CCU3 I created a Gewerk "Monitored" and added all channels of the room in question.
The host check added these channels in its output, but how to add the values in check_mk? My goal is to monitor a temperature or humidity in a check_mk graph and perhaps invoke warn/err notifications. What am I missing? Am I on the right track?

OMD[prod2]:~$` cmk -d ccu3-webui.jd.cretis.de | tail -31
<<<homematic:sep(59)>>>
UG Buero Fenster O:1;HSSTYPE;HMIP-SWDO
UG Buero Fenster S:1;HSSTYPE;HMIP-SWDO
UG Buero Fenster SO:1;HSSTYPE;HMIP-SWDO
UG Buero Heizung:1;HSSTYPE;HmIP-eTRV-2
UG Buero Heizung:1;ACTIVE_PROFILE;1;2019-01-09 13:28:50
UG Buero Heizung:1;ACTUAL_TEMPERATURE;21.700000;2019-01-09 13:28:50
UG Buero Heizung:1;LEVEL;0.210000;2019-01-09 13:28:50
UG Buero Heizung:1;SET_POINT_TEMPERATURE;22.000000;2019-01-09 13:28:50
UG Buero Heizung:1;VALVE_STATE;4;2019-01-09 13:28:50
UG Buero Heizungssteuerung:0;HSSTYPE;HmIP-HEATING
UG Buero Heizungssteuerung:1;HSSTYPE;HmIP-HEATING
UG Buero Heizungssteuerung:1;ACTIVE_PROFILE;1;2019-01-09 13:28:50
UG Buero Heizungssteuerung:1;ACTUAL_TEMPERATURE;21.700000;2019-01-09 13:28:50
UG Buero Heizungssteuerung:1;LEVEL;0.210000;2019-01-09 13:28:50
UG Buero Heizungssteuerung:1;PARTY_SET_POINT_TEMPERATURE;4.500000;2019-01-09 12:43:34
UG Buero Heizungssteuerung:1;PARTY_TIME_END;2000_01_01 00:00;2019-01-09 12:43:34
UG Buero Heizungssteuerung:1;PARTY_TIME_START;2000_01_01 00:00;2019-01-09 12:43:34
UG Buero Heizungssteuerung:1;SET_POINT_TEMPERATURE;22.000000;2019-01-09 13:28:50
UG Buero Heizungssteuerung:1;VALVE_STATE;4;2019-01-09 13:28:50
UG Buero Heizungssteuerung:3;HSSTYPE;HmIP-HEATING
UG Buero Heizungssteuerung:4;HSSTYPE;HmIP-HEATING
UG Buero Thermostat:1;HSSTYPE;HmIP-WTH-2
UG Buero Wandtaster:1;HSSTYPE;HMIP-WRC2
UG Buero Wandtaster:2;HSSTYPE;HMIP-WRC2
PEQ1841315;DESCRIPTION;{}
PEQ1841315;DUTY_CYCLE;1
PEQ1841315;DEFAULT;1
PEQ1841315;TYPE;CCU2
PEQ1841315;FIRMWARE_VERSION;3.4.8
PEQ1841315;CONNECTED;1

**Comment by alexreinert** [comment]
Can you please provide a screenshot of your Gewerk?
You need only to add the HmIP-WTH-2, as all of the other types are not supported right now. But I'm a little confused, that there is only the type description of your HmIP-WTH-2, but no datapoints at all. This could happen, if you use the wrong channel of the sensor or if you restarted your CCU and it has no values yet.

**Comment by joda123** [comment]
Hi Alexander,

I did the following steps:
(1) I removed all devices except HmIP-WTH-2 from the Gewerk "Monitored" as of your recommendation.
(2) reboot ccu3
(3) wait, until ccu3 is available again
(4) change set point temperature at HmIP-WTH-2 from 22.0 to 22.5 degrees and check, that the new set point temperature is displayed in webgui of ccu3 -> ok
(5) OMD[prod2]:~$ cmk -d ccu3-webui.jd.cretis.de | tail -12 
<<<homematic:sep(59)>>>
UG Buero Thermostat:1;HSSTYPE;HmIP-WTH-2
UG Buero Thermostat:1;ACTIVE_PROFILE;1;2019-01-09 14:03:06
UG Buero Thermostat:1;ACTUAL_TEMPERATURE;21.700000;2019-01-09 14:03:07
UG Buero Thermostat:1;HUMIDITY;31;2019-01-09 14:03:06
UG Buero Thermostat:1;SET_POINT_TEMPERATURE;22.500000;2019-01-09 14:03:06
PEQ1841315;DESCRIPTION;{}
PEQ1841315;DUTY_CYCLE;1
PEQ1841315;DEFAULT;1
PEQ1841315;TYPE;CCU2
PEQ1841315;FIRMWARE_VERSION;3.4.8
PEQ1841315;CONNECTED;1

-> ACTUAL_TEMPERATURE and HUMIDITY is now part of the output and a new service was detected by check_mk: "Humidity UG Buero Thermostat:1" -> perfect!
Perhaps I should have been more patient and wait for first values to come. This may take several minutes after a reboot. Temperatures don't change that often.
-> Issue solved! Thank you very much for your quick support!
Regards, Joachim
